cgi-bin/DW/User/Rename.pm

```diff
@@ -57,34 +58,36 @@ Return true if this user can be renamed to the given username
 
 Optional arguments are:
 =item force     => bool, default false
-=item errref    => array ref of errors
+=item errors    => DW::FormErrors object
```

Comment:
Slight preference to `errobj` in these kinds of things, just so it's clear it's an object. (You don't have to change things unless you want, but just a note...)